From b853fe9ea662a7e75fc0632fd1dc417297676103 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Mon, 13 Jul 2020 02:42:28 +0200
Subject: [PATCH] arm64: dts: sun50i-a64-pinephone: Enable HDMI audio on
 PinePhone

Now that HDMI audio patches are available, enable support for
PinePhone.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index 9fac2db918bc..9c1a5cbff472 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -262,6 +262,14 @@ &ehci1 {
 	status = "okay";
 };
 
+&hdmi_sound {
+	status = "okay";
+};
+
+&i2s2 {
+	status = "okay";
+};
+
 &hdmi {
 	hvcc-supply = <&reg_dldo1>;
 	status = "okay";
-- 
GitLab

